client/src/actions/todos.js

```diff
@@ -0,0 +1,62 @@
+export const FETCH_TODOS_BEGIN = 'FETCH_TASKS_BEGIN';
+export const FETCH_TODOS_SUCCESS = 'FETCH_TODOS_SUCCESS';
```

Comment:
define this in actions in similar auth structure

Reply:
These variables are inside actions. The only difference is I haven't used `createAction()`